OPAL/tac/src/main/scala/org/opalj/tac/Stmt.scala

```diff
@@ -77,14 +79,23 @@ sealed abstract class Stmt[+V <: Var[V]] extends ASTNode[V] {
     def asExprStmt: ExprStmt[V] = throw new ClassCastException();
     def asCaughtException: CaughtException[V] = throw new ClassCastException();
     def asCheckcast: Checkcast[V] = throw new ClassCastException();
+    def isCheckcast: Boolean = false;
 
     def isAssignment: Boolean = false
+    def isArrayStore: Boolean = false
     def isExprStmt: Boolean = false
     def isNonVirtualMethodCall: Boolean = false
     def isVirtualMethodCall: Boolean = false
     def isStaticMethodCall: Boolean = false
     def isCaughtException: Boolean = false
 
+    def isIf: Boolean = false
```

Comment:
Why is this offset with a blank line? Remove and sort these methods the same way the `as` methods are sorted above.